apps/frontend/src/network/users/__tests__/UserProfile.test.tsx

```diff
@@ -260,6 +260,41 @@ describe('a header edit button', () => {
     );
   });
 
+  it('can open and close   personal info remaining on the tab', async () => {
+    const userProfile: UserResponse = {
+      ...createUserResponse(),
+      biography: 'My Bio',
+    };
+
+    const {
+      getByText,
+      getByTitle,
+      queryByText,
+      findByLabelText,
+      findByText,
+    } = await renderUserProfile(userProfile);
+
+    // Open and close on research tab
```

Comment:
```suggestion
    // Open and close on research tab
    userEvent.click(await findByText(/research/i, { selector: 'nav *' }));
```
just because otherwise this also implicitly tests what the default tab is